final_prep/annotations.rb

```diff
@@ -16,28 +20,51 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  #This is returning the attributes in the hash model assigned to the 'built_bear' variable.
   return built_bear
+  #This ends the funciton(method)
 end
-
+#This is calling the 'build_a_bear' method and passing the arguments though the method.
+#It is also passing an array with three string elements through the method.
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
+#This is calling the 'build_a_bear' method and passing arguments through the method.
+#This is also passing an array with two string elements through the method.
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 # FizzBuzz
-
+# This is creating a method with three arguments.
 def fizzbuzz(num_1, num_2, range)
+  #This is creating a loop with the .each method.
+  #There is a range set with the argument rannge, and the the '|i|' is telling the method to repeat the metod for every number in the range.
   (1..range).each do |i|
+  #This is is saying if the range number divided by the argument given has a remainder of 0,
+  # AND also has a remainder of 0 when devided by the second argument to print 'fizzbuzz'.
```

Comment:
Nice explanations of your loop and modulo in plain English! Strong job overall with your annotations.